<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartyDetailsForm</class>
 <widget class="QWidget" name="PartyDetailsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Party Contact Information</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="addressGroupBox">
        <property name="title">
         <string>Address</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="firstLabel">
           <property name="text">
            <string>First Name</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="lastLabel">
           <property name="text">
            <string>Last Name</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="middleLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="firstLineEdit"/>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="middleLabel">
           <property name="text">
            <string>Middle</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="lastLineEdit"/>
         </item>
         <item row="4" column="0" colspan="3">
          <widget class="QWidget" name="widget" native="true">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="streetLabel">
              <property name="text">
               <string>Street</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLabel" name="statelabel">
              <property name="text">
               <string>State</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="countyLabel">
              <property name="text">
               <string>County</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="countyLineEdit"/>
            </item>
            <item row="2" column="3">
             <widget class="QLineEdit" name="stateLineEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="cityLabel">
              <property name="text">
               <string>City</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="streetLineEdit"/>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="cityLineEdit"/>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="unitLabel">
              <property name="text">
               <string>Unit</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="zipLabel">
              <property name="text">
               <string>Zip</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QLineEdit" name="zipLineEdit"/>
            </item>
            <item row="0" column="3">
             <widget class="QLineEdit" name="unitLineEdit"/>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="phoneGroupBox">
        <property name="title">
         <string>Phone</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="4">
          <widget class="QLineEdit" name="workExtLineEdit"/>
         </item>
         <item row="2" column="4">
          <widget class="QLineEdit" name="mobileExtLineEdit"/>
         </item>
         <item row="1" column="4">
          <widget class="QLineEdit" name="homeExtLineEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="homeLineEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="mobileLabel">
           <property name="text">
            <string>Mobile</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QLabel" name="workExtLabel">
           <property name="text">
            <string>Ext</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="homeLabel">
           <property name="text">
            <string>Home</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="workLabel">
           <property name="text">
            <string>Work</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="homeExtLabel">
           <property name="text">
            <string>Ext</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="mobileExtLabel">
           <property name="text">
            <string>Ext</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="mobileLineEdit"/>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="workLineEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="emailFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QLabel" name="emailLabel">
           <property name="text">
            <string>Email</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="emailLineEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="miscFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="0" column="0">
          <widget class="QLabel" name="numInHomeLabel">
           <property name="text">
            <string># In Home</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="numInHomeLineEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="attorneyLabel">
           <property name="text">
            <string>Attorney</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="attorneyLineEdit"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
